lib/index.js

```diff
@@ -75,7 +76,7 @@
         /* global document */
         var $ = modules.$;
         var Assertion = chai.Assertion;
-        Assertion.addMethod('accessible', function(config) {
+        Assertion.addMethod('accessible', function(config, ignoreGlobalRules) {
```

Comment:
by default, global rules should be ignored... perhaps, we can have `runGlobalRules` param. This is because, mostly the assertion will run on a DOM element